docs/ops.html

```diff
@@ -229,6 +229,63 @@ <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing C
   Deletion of requested consumer groups ('my-group', 'my-other-group') was successful.
   </pre>
 
+  <p>
+  To reset offsets of a consumer group to latest offset, first make sure the instances are inactive, then:
+
+  <pre class="brush: bash;">
+  &gt; bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --reset-offsets --group consumergroup1 --topic topic1 --to-latest
+
+  TOPIC                          PARTITION  NEW-OFFSET
+  topic1                         0          0
+  </pre>
+
+  --reset-offsets supports one consumer group at the time.
+  <p>
+  It has 3 execution options:
+  <ul>
+    <li>
+      (default) to plan which offsets to reset.
+    </li>
+    <li>
+      --execute : to execute --reset-offsets process.
+    </li>
+    <li>
+      --export : to export the results to a CSV format.
+    </li>
+  </ul>
+
+  --reset-offsets also has following scenarios to choose from (atleast one scenario must be choosen):
+  <ul>
+    <li>
+      --to-datetime &lt;String: datetime&gt; : Reset offsets to offsets from datetime. Format: 'YYYY-MM-DDTHH:mm:SS.sss'
+    </li>
+    <li>
+      --to-earliest : Reset offsets to earliest offset.
+    </li>
+    <li>
+      --to-latest : Reset offsets to latest offset.
+    </li>
+    <li>
+      --shift-by &lt;Long: number-of-offsets&gt; : Reset offsets shifting current offset by 'n', where 'n' can be positive or negative.
```

Comment:
We should probably mention above that out of range offsets will be adjusted to the log start and log end offsets. For example, if the log end offset is 10 and the offset after shifting is 15, then we will actually commit offset 10. Similarly for the other options.

Reply:
Added a general note at the end of this section.